queries/parse.go

```diff
@@ -76,21 +76,32 @@ func AppendMatcher(queryValues url.Values, queryValuesForAuth url.Values, key st
 		}
 	}
 	matcher := &labels.Matcher{
-		Name:  key,
+		Name:  authKey,
 		Type:  labels.MatchRegexp,
 		Value: LabelValuesToRegexpString([]string{value}),
 	}
 	err := InjectMatcher(queryValuesForAuth, matcher)
-	return err
+	return authKey, value, err
 }
 
-func ParseAuthorizations(queryValues url.Values) url.Values {
+func ParseAuthorizations(queryValues url.Values) (url.Values, []string, []string) {
 	queryValuesForAuth := make(url.Values)
 
-	AppendMatcher(queryValues, queryValuesForAuth, "cluster", "all clusters")
-	AppendMatcher(queryValues, queryValuesForAuth, "namespace", "all namespaces")
+	var keys []string
+	var values []string
+	cluster_key, cluster, _ := AppendMatcher(queryValues, queryValuesForAuth, "cluster", "cluster", "all clusters")
```

Comment:
Why are we ignoring the error code here?

Reply:
@larsks I can make a followup PR to improve the error handling here.

Reply:
I can also add the code comments in a follow up PR.